pkg/psmdb/mongo/models.go

```diff
@@ -74,6 +74,8 @@ type WriteConcern struct {
 const (
 	envMongoDBClusterAdminUser     = "MONGODB_CLUSTER_ADMIN_USER"
 	envMongoDBClusterAdminPassword = "MONGODB_CLUSTER_ADMIN_PASSWORD"
+	envMongoDBUserAdminUser        = "MONGODB_USER_ADMIN_USER"
+	envMongoDBUserAdminPassword    = "MONGODB_USER_ADMIN_PASSWORD"
 )
```

Comment:
no need in this consts anymore

Reply:
removed https://github.com/percona/percona-server-mongodb-operator/pull/387/commits/6000f694034c0beec304d71c3877278604ad560f